js-miniapp-sdk/src/miniapp.ts

```diff
@@ -181,6 +183,7 @@ export class MiniApp implements MiniAppFeatures, Ad, Platform {
   galleryManager: GalleryBridge = new GalleryBridge();
   webviewManager: WebviewManager = new WebviewManager();
   esimService: Esim = new Esim();
+  appSettingsLaunch: AppSettingsLaunch = new AppSettingsLaunch();
```

Comment:
I think we don't need to create a new module just for this interface. I feel we can reuse miniappUtils